### PR TITLE
update to nco 4.6.8

### PR DESCRIPTION
**Pull request by xylar**
Update the feedstock to nco 4.6.8

### CODE REVIEW

**Comment by xylar** [comment]
@ocefpaf, I'm new to conda-forge but did my best to follow previous updates here.  I ran docker on my own machine (running ubuntu 16.04) and it seemed to work as expected (modulo some warnings about conflicting versions of libgfortran).

My group and I use nco for a lot of our work and we'd be very keen to have new features related to the `ncclimo` tool available to us on conda-forge.  So I figured I'd try my hand at updating the feedstock.

**Comment by ocefpaf** [comment]
The PR is perfect it is just odd that the CIs did not start :unamused: 

Let me look into that and I'll merge this ASAP.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ requirements:
     - expat
     - libgcc
     - esmf
-    - openblas 0.2.19|0.2.19.*
+    - openblas 0.2.20|0.2.20.*
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Even though `0.2.20` is available we are not pinning to that version b/c we could not build `scipy` with it.
To ensure ecosystem stability we should revert this to `openblas 0.2.19|0.2.19.*`.

PS: the pinning info is located [here](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L39-L86).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ requirements:
     - krb5 1.14.*
     - texinfo
     - bison
-    - flex 2.6.0
+    - flex 2.6.3
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I don't recall why we had this pinned to `2.6.0`, but if `2.6.3` works we may as well as leave this unpinned.

**Comment by ocefpaf** [comment]
It seems that GitHub's hooks are experiencing some [delays](https://gitter.im/conda-forge/conda-forge.github.io?at=59b1c5d419147ac323167d44).

File: recipe/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipe/meta.yaml]: Thank you for pointing me to that script.  I now see how the pinned versions are chosen.

File: recipe/meta.yaml (hunk 2 above)
Comment (xylar) [reply recipe/meta.yaml]: Okay, I have unpinned flex in my latest commit.  Let's see if it passes CI.

**Comment by xylar** [comment]
@ocefpaf, Travis CI really doesn't seem to want to run.  Is there anything we can do to "bump" it?  I haven't had any trouble with my Travis CI on other repos today.  Thanks!

**Comment by ocefpaf** [comment]
Not really. Travis-CI had a major outage, see https://www.traviscistatus.com/incidents/4qylrqvy50gy , all we can do is wait.

If you are not in a hurry for the OS X packages we can merge this, cash the Linux package, and monitor the Travis-CI build on `master`.

**Comment by xylar** [comment]
No worries.  I see.  We're probably not texting our repo on OSX so that's probably why it's going through.  Sounds like it would be better to just wait until things get resolved.  Thanks again.

**Comment by ocefpaf** [comment]
Let's get this in, it seems that Travis-CI will take a while to get back online.

Thanks @xylar! Consider adding yourself as a maintainer for this recipe :wink:

**Comment by xylar** [comment]
@ocefpaf, I'll definitely consider adding myself as a maintainer next time I make a PR.  Thanks for the invitation and thanks for merging.

**Pull request merged**